### DISCUSSION TITLE
Is there a way to make spacedrive as default file manager?

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [spacedriveapp/spacedrive#2786] [Community Support]
I find spacedrive really better and helpful than the windows file manager and if I want any additional option in it I could contribute to it. I choose it because of the ascending and descending feature I wanted in windows file manager which is the worst. Now I am asking this so that I can make it my default file manager. 

Coming after reading this one #2402.

**Comment by Au-angeloupali** [comment]
@Gitstar-OC I can't tell if the project is on a dead track, plus no updates, no replies to discussions
